flake.nix

```diff
@@ -304,6 +317,14 @@ rec {
               export PATH_PILLS="${pills}"
               export PATH_DEMOS="${demos}"
 
+              if [ -d @NixOS/branding ]; then
+                rm -rf @NixOS/branding
+              fi
+              mkdir -p @NixOS/branding
+              cp -R --no-preserve=mode,ownership ${
+                branding.hydraJobs.nixos-branding.${system}.npm-package
```

Comment:
Same as the other comment

> Rather than accessing `npm-package` directly through the flake input, wouldn't it be better to use the expose default overlay? This would add `nixos-branding` to the top-level of the import package set and then you could access `npm-package` via `nixos-branding.npm-package`.